apps/balancer-tools/seed.ts

```diff
@@ -262,6 +268,81 @@ async function extractGauges() {
   await processGauges(response.data);
 }
 
+async function extractGaugesSnapshot() {
```

Comment:
This gets the relative weight of each gauge every day since the gauge was created. This is getting one by one, so no need to worry about rate limit. Also, this can be a multicall to be faster, but definitely not a must

Reply:
is this idempotent?